client/app/services/query.js

```diff
@@ -349,10 +349,11 @@ export class Parameter {
   }
 
   toUrlParams() {
+    const prefix = this.urlPrefix;
     if (this.isEmpty) {
-      return {};
+      return { [`${prefix}${this.name}`]: null };
```

Comment:
@gabrieldutra can you explain this code change?

Reply:
Yes, a {} does nothing, while setting a null value on that will remove the param from the URL in case it exists (I introduced this logic with Dynamic Date/Date Ranges).